packages/orbit-components/src/Tabs/components/Tab/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { convertHexToRgba } from "@kiwicom/orbit-design-tokens";
+
+import { useTabs, useTab } from "../../TabContext";
+import type { Props } from "./types";
+import { TYPE_OPTIONS } from "./consts";
+import defaultTheme from "../../../defaultTheme";
+
+const getColorBackground = ({ type, theme }) => {
+  if (type === TYPE_OPTIONS.MEDIUM) return theme.orbit.backgroundColorTabBundleMedium;
+  if (type === TYPE_OPTIONS.BASIC) return theme.orbit.backgroundColorTabBundleBasic;
+
+  return theme.orbit.backgroundColorTabBundleTop;
+};
+
+const getBackgroundType = (state?: "hover" | "active") => ({
+  theme,
+  $type,
+}: {
+  theme: typeof defaultTheme;
+  $type: Props["type"];
+}) => {
+  if (state === "hover") {
+    if ($type === TYPE_OPTIONS.MEDIUM) return convertHexToRgba(theme.orbit.paletteBundleMedium, 8);
+    if ($type === TYPE_OPTIONS.BASIC) return convertHexToRgba(theme.orbit.paletteBundleBasic, 8);
+    return theme.orbit.paletteWhiteHover;
+  }
+
+  if (state === "active") {
+    if ($type === TYPE_OPTIONS.MEDIUM) return convertHexToRgba(theme.orbit.paletteBundleMedium, 12);
+    if ($type === TYPE_OPTIONS.BASIC) return convertHexToRgba(theme.orbit.paletteBundleBasic, 12);
+
+    return theme.orbit.paletteWhiteActive;
+  }
+
+  return "none";
+};
+
+const StyledTab = styled.button<{
+  $type: Props["type"];
+  active?: Props["active"];
+  compact: boolean;
+}>`
+  ${({ theme, active, $type, compact, disabled }) => css`
+    display: flex;
+    border: 0;
+    position: relative;
+    appearance: none;
+    flex-direction: row;
+    justify-content: space-between;
+    box-sizing: border-box;
+    align-items: center;
+    font-family: ${theme.orbit.fontFamily};
+    cursor: ${!disabled && "pointer"};
+    opacity: ${disabled && "0.5"};
+    padding: ${compact ? "5px 16px" : "9px 16px"};
```

Comment:
I would have done it like this. With this, the content is actually centrally aligned with the full height of the Tab, like in Figma. But I am not sure if it is what is intended, or if it should be centrally aligned with the white part. Therefore, I'll leave the suggestion but it is up to you, non-blocking.
```suggestion
    padding: ${compact ? "6px 16px 4px" : "10px 16px 8px"};
```